lib/watcher.js

```diff
@@ -6,6 +6,7 @@ const chokidar = require('chokidar');
 const flatten = require('arr-flatten');
 const union = require('array-union');
 const uniq = require('array-uniq');
+const chalk = require('chalk');
```

Comment:
You should require chalk from `./chalk`. That way you get a properly configured instance.